src/client/opamCommands.ml

```diff
@@ -303,11 +303,27 @@ let init cli =
        $(b,--config) is used). Use this at your own risk, without sandboxing \
        it is possible for a broken package script to delete all your files."
   in
+  let cygwin_internal =
+    mk_vflag ~cli `none [
+      cli_from cli2_2, `internal, ["cygwin-internal-install"],
+      "Let opam setup and manage an internal Cygwin install";
+      cli_from cli2_2, `default_location, ["cygwin-local-install"],
+      "Use preexistent Cygwin install";
+      cli_from cli2_2, `no, ["no-cygwin-setup"],
+      "Don't setup Cygwin";
+    ]
+  in
+  let cygwin_location =
+    mk_opt ~cli (cli_from cli2_2) ["cygwin-location"] "DIR"
+      "Specify Cygwin root location"
+      Arg.(some dirname) None
+  in
```

Comment:
Other proposal: `--cygwin [no|internal|default|<location>]`

Reply:
Hm I like the proposal (but it's ok as is)

Reply:
now, we do